exalearn-api/ExaLearn.WebApi/Controllers/QuestionController.cs

```diff
@@ -19,6 +19,24 @@ public QuestionController(IQuestionService questionService)
             _questionService = questionService;
         }
 
+        [HttpPost("createGrammar")]
```

Comment:
try to add from swagger 3 type of question

Reply:
added